viafabric-mc120/src/main/resources/fabric.mod.json

```diff
@@ -40,7 +40,7 @@
   },
   "depends": {
     "fabric-resource-loader-v0": "*",
-    "minecraft": [">=1.20.3"],
```

Comment:
There are no changes in these versions, I don't see a reason to limit to 1.20.4 only

Reply:
I would usually say the same, but in this case it's a bit useless to drop a version for a single line change, so /shrug